src/main/java/com/minecolonies/coremod/placementhandlers/PlacementHandlers.java

```diff
@@ -65,39 +66,43 @@ private PlacementHandlers()
         handlers.add(new ChestPlacementHandler());
         handlers.add(new WayPointBlockPlacementHandler());
         handlers.add(new GeneralBlockPlacementHandler());
-
     }
 
     //If he woudln't count the bracket spaces we'd be under 25 easily.
     public static class FireplacementHandler implements IPlacementHandler
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this class from 36 to the maximum allowed 30 or externalize it in a public class. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2972)